bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -103,6 +103,18 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="netatmo:weather">
+		<parameter name="id" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
+			<label>@text/config.equipmentId.label</label>
+			<description>@text/config.equipmentId.description</description>
+		</parameter>
+		<parameter name="refreshInterval" type="integer" min="20" unit="s">
```

Comment:
Does it make sense to allow 20 s? It seems almost like a guarantee to get banned. 🙂

Reply:
That's true. It has been like this since a while.

Reply:
Changed